agent/functional_tests/testdata/taskdefinitions/awslogs-secret-support-multiline/task-definition.json

```diff
@@ -0,0 +1,26 @@
+{
+  "family": "test-awslogs-secret-multiline",
+  "executionRoleArn": "$$$EXECUTION_ROLE$$$",
+  "containerDefinitions": [{
+    "essential": true,
+    "memory": 20,
```

Comment:
low memory had been a problem - https://github.com/aws/amazon-ecs-agent/pull/1842. can you change to 64?